connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -56,11 +58,29 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
     {
       maxRequestsPerCrawl: MAX_PAGES,
       maxConcurrency: CONCURRENCY,
-
+      maxRequestsPerMinute: 60, // 5 requests per second to avoid overloading the target website
+      requestHandlerTimeoutSecs: REQUEST_HANDLING_TIMEOUT,
```

Comment:
Shouldn't this be 120s?

Reply:
no the timeout should include the whole time of request handling, 120s for getting the page + parsing, then 300s which is the upsert timeout

Reply:
a follow-up  PR could be that we move the upserts to separate activities, so we would  have 1 crawling activity that would move pretty fast, and many (up to 512 aka MAX PAGES) upsert activies that would last 1 to 500 mns

In that case reducing the timeout to 120s would indeed be adequate

Reply:
Roger that